memory/src/lib.rs

```diff
@@ -116,26 +123,44 @@ where
         // Sort first by addr, then by clk
         ops.sort_by_key(|(clk, op)| (op.get_address(), *clk));
 
-        // Consecutive sorted clock cycles for an address should differ no more
-        // than the length of the table (capped at 2^29)
-        Self::insert_dummy_reads(&mut ops);
+        // // Consecutive sorted clock cycles for an address should differ no more
+        // // than the length of the table (capped at 2^29)
+        // Self::insert_dummy_reads(&mut ops);
```

Comment:
```suggestion
       // Consecutive sorted clock cycles for an address should differ no more
       // than the length of the table (capped at 2^29)
        // Self::insert_dummy_reads(&mut ops);
```
Did you mean to comment out this?

Reply:
I guess so otherwise there will be a warning. Feel free to ignore this.